docs/clients/promtail/stages/match.md

```diff
@@ -46,40 +50,48 @@ pipeline_stages:
 - labels:
     app:
 - match:
-    selector: "{app=\"loki\"}"
+    selector: '{app="loki"}'
     stages:
     - json:
         expressions:
           msg: message
 - match:
     pipeline_name: "app2"
-    selector: "{app=\"pokey\"}"
+    selector: '{app="pokey"}'
     stages:
     - json:
         expressions:
           msg: msg
+- match:
+    selector: '{app="promtail"} |~ ".*noisy error.*"'
+    drop_entries: true
```

Comment:
What do you think about switching it to `action: drop` (defaults to `action: keep`), in order to keep the way open to further actions?

Reply:
Yes good idea !